strawberryfields/ops.py

```diff
@@ -23,7 +23,7 @@
 import numpy as np
 
 from scipy.linalg import block_diag
-from scipy.special import factorial as fac
+import scipy.special as ssp
```

Comment:
How come this was changed?

Reply:
I think most style guides recommend importing entire modules over importing stuff from modules, as it makes the code more readable. `factorial` is only used in one method so it does not even add much extra characters.

https://docs.python-guide.org/writing/structure/#modules